bpm/bonita-core/bonita-process-engine/src/test/java/org/bonitasoft/engine/bpm/contract/validation/ContractStructureValidatorTest.java

```diff
@@ -52,17 +54,17 @@
 public class ContractStructureValidatorTest {
 
     @Mock
-    private TechnicalLoggerService logger;
-
-    @Mock
     ContractTypeValidator typeValidator;
-
+    @Mock
+    private TechnicalLoggerService logger;
     @InjectMocks
     private ContractStructureValidator validator;
 
     @Before
     public void setUp() {
         when(logger.isLoggable(ContractStructureValidator.class, DEBUG)).thenReturn(true);
+        doReturn(true).when(typeValidator).validate(any(SInputDefinition.class), anyObject(), any(ErrorReporter.class));
+
```

Comment:
No I can't, it return false otherwise and fore these tests I the default return of type validator to be true